traffic_portal/package.json

```diff
@@ -50,12 +41,12 @@
     "angular-route": "1.8.0",
     "angular-sanitize": "1.8.0",
     "angular-ui-bootstrap": "0.14.3",
-    "bootstrap-sass": "3.3.6",
-    "es5-shim": "4.5.6",
+    "bootstrap-sass": "3.4.1",
+    "es5-shim": "4.6.4",
     "flot": "2.3.2",
-    "font-awesome": "4.5.0",
+    "font-awesome": "4.7.0",
     "jquery": "3.6.0",
     "jquery.flot.tooltip": "0.9.0",
-    "json3": "3.3.2"
+    "json3": "3.3.3"
```

Comment:
The description for the `json3` package on `npm` starts (after a deprecation notice) with the words _"JSON 3 **was**..."_ (emphasis mine), which is never a good sign. Neither is:

![image](https://user-images.githubusercontent.com/6013378/150589675-05a24803-2791-478f-b03d-24938e1ddfba.png)

No major browser version since 2010 has not had support for `JSON` in the global scope ([source](https://caniuse.com/?search=JSON)) - again, predating the existence of Traffic Portal

Reply:
This is also referenced somewhere like `es5-shim`, so it should also be removed in another PR